packages/create-redwood-app/template/.vscode/extensions.json

```diff
@@ -8,7 +8,8 @@
     "pflannery.vscode-versionlens",
     "editorconfig.editorconfig",
     "prisma.prisma",
-    "graphql.vscode-graphql"
+    "graphql.vscode-graphql",
+    "firsttris.vscode-jest-runner"
```

Comment:
I'm not sure about this recommendation. I personally don't think any of the free jest extensions are any good. Wallaby is the one I'd like to recommend, but its not free or open source.

Reply:
Does the debugging setting work with the official Jest extension by any chance?

Reply:
> I'm not sure about this recommendation. I personally don't think any of the free jest extensions are any good. Wallaby is the one I'd like to recommend, but its not free or open source.

What's not good about it? Only additional feature I'd like is "profile" -- which I know wallaby has. Regardless something that works seems better than nothing.

Reply:
> Does the debugging setting work with the official Jest extension by any chance?

It works. You mean the one published by "orta" right? I'm not sure how to actually verify what is the "real" one here -- but IIRC that is the one.

<img width="883" alt="image" src="https://user-images.githubusercontent.com/9519712/167020956-2987410c-bac0-41ad-b560-734f238fdb07.png">

Reply:
^ this one
<img width="400" alt="image" src="https://user-images.githubusercontent.com/9519712/167021099-360543a1-6d00-4feb-9521-9cacc9e56bbf.png">

Reply:
I don't care what extension we use -- happy to change it if desired. This is a change that is easy to reverse with no consequence (just a rec) so there is no downside and all upside. In those cases, let's just pick one and forge ahead.

Reply:
I'm of the same mind as @dac09. I'm hesitant to recommend an extension. Configuring `launch.json` and adding debugging options to the CLI commands that need them like we did for `yarn rw dev` in this PR here (https://github.com/redwoodjs/redwood/pull/4904) seems like the way to go.